### PR TITLE
Add phpmyadmin service as optional

### PR DESCRIPTION
**Pull request by tesshsu**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add a phpMyAdmin service to Docker compose file in order to easily access BO from [ Idea] #32425
| Type?             | improvement
| Category?         | IN
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | Indicated in document, run docker-compose -f docker-compose.yml -f docker-composer.override.yml up to check if prestashop is correctly installed with phpMyAdmin
| Fixed ticket?     | [ Idea] #32425

### CODE REVIEW

File: docker-composer.override.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+version: '2.1'
```
Comment (FabienPapet) [inline docker-composer.override.yml]: Can you please rename this file to `docker-compose.override.yml.dist` ?

File: README.md [hunk 2]
```diff
@@ -81,6 +81,11 @@ docker compose down -v
 docker compose build --no-cache
 docker-compose up --build --force-recreate
 ```
+**Note:** To add a PHPMyAdmin service to your Docker Compose setup, start your Docker Compose setup with the -f flag to include both the docker-compose.yml and docker-compose.override.yml files:
```
Comment (FabienPapet) [inline README.md]:
```suggestion
**Note:** To add a PHPMyAdmin service to your Docker Compose setup, you can copy the existing `docker-compose.override.yml.dist` to `docker-compose.override.yml` and then start your Docker Compose setup (override file will be included as it is a convention)
```

File: README.md [hunk 3]
```diff
@@ -81,6 +81,11 @@ docker compose down -v
...
+
+```
+docker-compose -f docker-compose.yml -f docker-composer.override.yml up
```
Comment (FabienPapet) [inline README.md]:
```suggestion
cp docker-compose.override.yml.dist docker-compose.override.yml
docker-compose up
```

**Review by FabienPapet — DISMISSED** [review]
Thank you for the PR, nice work already, I addressed some comments

**Comment by tesshsu** [comment]
@FabienPapet
Thanks for the feedback, updated by your recommends, any further request let me know, thank you

**Review by zuk3975 — APPROVED** [review]

**Review by tleon — APPROVED** [review]

File: docker-composer.override.yml
Comment (kpodemski) [inline docker-composer.override.yml:1]: @tesshsu I think we don't need this file by default

Comment (tesshsu) [reply docker-composer.override.yml:1]: Hi @kpodemski , yes, the file had been removed, check the commit

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
This requires QA by a dev. @FabienPapet could you check?

**Comment by tleon** [comment]
I tested it LGTM.

**Review by matks — APPROVED** [review]
Thank you @tesshsu nice contribution 😉

**Comment by FabienPapet** [comment]
Thank you @tesshsu for the PR and @tleon for testing it.

**Pull request merged**